src/plugins/management/public/plugin.ts

```diff
@@ -8,6 +8,7 @@
 
 import { i18n } from '@kbn/i18n';
 import { BehaviorSubject } from 'rxjs';
+import { SharePluginSetup, SharePluginStart } from 'src/plugins/share/public';
```

Comment:
:art: We're trying to use type imports whenever possible:

```suggestion
import type { SharePluginSetup, SharePluginStart } from 'src/plugins/share/public';
```